### DISCUSSION TITLE
Limited support for slurm?

### DISCUSSION THREAD

**Discussion by jianxinwang** [nextflow-io/nextflow#2329] [Q&A] [answered]
Our HPC uses the Slurm job schedular.  However, when I looked at the documentation for slurm in the executor section, it has very basic parameters we can set. Our HPC slurm setup uses the additional parameters like "clusters",  "account", "constraints", "qos" other than just the "partition" parameter (which called queue in nextflow). I put these extra parameters in the nextflow config file and tried to run a nextflow command. It did not work because when examining the resultant .command.run script, these extra parameters did not show up, only the basic slurm parameters. I wonder if this is caused by putting these parameters in the wrong location or current version of nextflow lacks the support for these parameters? Please help. Thanks!

**Comment by tbugfinder** [comment]
Could you paste some sample code you were using?

**Comment by jianxinwang** [comment, marked as answer]
I got the solution. I found I can put the other slurm parameters as a
single line in clusterOptions. Thanks!

Jason

**Reply by marcellobeltrami** [reply]
Hello, although I am not the OP I managed to get it to work using the following code snipped. Here I provide two options (one with full names, the other one with shortened names). I think it would be quite beneficial to have an example like this one on the actual NF documentation :). 

<clusterOptions = '-q default -t 00:30:00 -C cpu' > or <clusterOptions = '--qos=default --time=00:30:00'>
